### ISSUE TITLE
Camera Change Detection

### ISSUE THREAD

**Issue by programmeddeath1** [SilvioGiancola/SoccerNetv2-DevKit#3]
Hi, can you tell me how is the labels-camera.json file generated? 
I am assuming it is generated through inference run by BasicModel-Segmentation. Will it be run from the inference code for Task2-BasicModelSegmentation and if so can it be shared, it seems to be commented in the main.py for task2.

Thank you for your help in advance!
This seems to be a very interesting research and your loss function is amazing conceptually and even practically kudos!

**Comment by meisamjam** [comment]
Hi 
The label-camera.json files are annotations(ground-truths) that have been manually annotated by people. These ground-truths are then used to train the BasicModel-Segmentation, CALF-segmentation, and CALF-detection and to evaluate the performance of the different models.
